README.md

```diff
@@ -53,6 +53,10 @@ Once you've got all of your code into Carbon, you can customize your image by ch
 
 After you've customized your image you can either Tweet a link to the image, or save it directly.
 
+If you use the &apos;Tweet&apos; button, Carbon will automatically make your image accessible. However, if you want to manually tweet your carbon image, please check out [how to make your Twitter images accessible](https://help.twitter.com/en/using-twitter/picture-descriptions).
+
+If you include a carbon image in a post, the source code will be invisible for assistive technology, it will not be possible to enlarge it or copy it, etc. Please, think about adding another element with the source code as text. For example, you can add [a HTML Details Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) below the image.
```

Comment:
I believe for images an `alt` attribute is sufficient for making images screen-reader accessible, no? 🙂 I would love to encourage people to do that. 

Alternatively, users of Carbon can add `title` attributes to their images to also add screen-readability. Do you think this is an equivalent alternative?

Reply:
Hi @mfix22, I understand you, because I thought the same at the beginning, but we must open our mind to other types of disabilities and other types of assistive technology. 

That solution would be valid for screen readers. Furthermore, ARIA provides `aria-describedby` and `aria-details` for facilitating how to add a description to an image.

However, in this case, it's not a simple description. It's the text within the image. There are other visual disabilities for which text within images cannot be enlarged correctly, the font cannot be changed, the contrast cannot be changed, etc.

That's the reason why it's important to offer another element with the source code as text. 

It might appear difficult to take into account all the disabilities and all the types of assistive technology, although I think it's easy in this case. Text within an image is not accessible, it's only necessary to offer the source code as text in an additional element.

You have made a great work, because source code is beautiful with Carbon. I thought about a HTML Details Element, because it doesn't interfere too much with the visual aspect (its content is only visible when it's toggled into an "open" state). Maybe there are more options, but my goal is to raise awareness of it.

Thanks for your time, @mfix22 !